src/App.js

```diff
@@ -1,18 +1,23 @@
 import './App.css';
 import { Route, Routes } from 'react-router-dom';
-import { Cart, HomePage, MockAPI, ProductList, Wishlist, ProductDetails } from './pages/index';
+import { Cart, HomePage, MockAPI, ProductList, Wishlist, ProductDetails, PrivateRoute, SignUp, Login, UserProfile } from './pages/index';
 
 function App() {
   return (
     <div className="App">
       <Routes>
-        <Route path='/' element={<HomePage />}/>
```

Comment:
Inspite of defining all the rotes over here.You can create a sepaate file for mainitaing all the routes.This will keep your app.js more modular